docs/tables/aws_waf_traffic_log/index.md

```diff
@@ -138,6 +138,33 @@ partition "aws_waf_traffic_log" "my_logs_prefix" {
 }
 ```
 
+### Collect logs from a CloudWatch log group
+
+Collect CloudTrail logs stored in a CloudWatch log group.
+
+```hcl
+partition "aws_waf_traffic_log" "my_logs" {
+  source "aws_cloudwatch_log_group" {
+    connection = connection.aws.logging_account
+    log_group_name = "aws-waf-log-group"
```

Comment:
Is it common to use a log prefix when storing these logs? If so, should we just have 1 example that uses `log_stream_prefix`?

Reply:
Always the logs are stored in the log stream, if an user doesn't specify the `log_stream_prefix` in the config then the CloudWatch source will pull the logs from all the log streams available in the specified log group. We have a example configuration using `log_stream_prefix` with title `Collect logs from a CloudWatch log group with a log stream prefix`